bots/pr/src/main/java/org/openjdk/skara/bots/pr/AuthorCommand.java

```diff
@@ -105,14 +105,13 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
                     }
                 }
                 if (currAuthor.isEmpty()) {
-                    reply.println("There is no author set in this pull request.");
+                    reply.println("There is no overriding author set in this pull request.");
                 } else {
                     if (currAuthor.get().equals(author.get())) {
-                        reply.println(Authors.removeAuthorMarker(author.get()));
-                        reply.println("Author `" + author.get() + "` successfully removed.");
+                        reply.println(OverridingAuthor.removeAuthorMarker(author.get()));
+                        reply.println("Overriding author `" + author.get() + "` was successfully removed. When this pull request is integrated, the pull request author will be the author of the commit.");
```

Comment:
Reading this again, it can be misinterpreted.
```suggestion
                        reply.println("Overriding author `" + author.get() + "` was successfully removed. When this pull request is integrated, the pull request author will be used as the author of the commit.");
```